### DISCUSSION TITLE
From a Beginner player just getting into Beatoraja and bms as a whole: THANK YOU!

### DISCUSSION THREAD

**Discussion by sigonasr2** [wcko87/beatoraja-english-guide#38] [Guide discussion]
This guide is about as comprehensive as it can get for a new player to bms and beatoraja. Got everything setup just fine and even got all my lingering questions answered (like accidently hiding songs, figuring out green number, skinning, ideal configuration settings, the difficulty tables, hold note types). It's all here. So thanks for all of this, I'm having a great time with the game thanks to this guide!

**Comment by wcko87** [comment]
:)
